Yoast/YoastSEO.js
# Adds getLinks to researcher

## Summary

This PR can be summarized in the following changelog entry:
Enables the 'getLinks' research to be accessed via the researcher. 

## Relevant technical choices:

* Added the get links research to the researcher, and made sure it accepts the paper as its argument, so it can be run like all other researchers. 

## Test instructions

This PR can be tested by following these steps:

* Make sure all tests pass for YoastSEO.js
* In  WordPress, run `YoastSEO.app.researcher.getResearch( "getLinks" )` in the console. It should not give errors, but display an array of all links in the post.

This is a requirement for https://github.com/Yoast/wordpress-seo-premium/issues/875
